### PR TITLE
Feature feedback

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/Redux/actionCreators/ServiceActions.js [hunk 1]
```diff
@@ -43,7 +44,7 @@ export const fetchServiceSuccess = res => dispatch => {
 
 export const fetchService = (pagination, filters = []) => async dispatch => {
   dispatch(loaderActions.startAIServiceListLoader);
-  let url = new URL(`${APIEndpoints.GET_SERVICE_LIST.endpoint}/service`);
+  let url = new URL(`${APIEndpoints.CONTRACT.endpoint}/service`);
```
Comment (raamb) [inline src/Redux/actionCreators/ServiceActions.js]: Can be const?

File: src/Redux/actionCreators/ServiceActions.js [hunk 2]
```diff
@@ -121,3 +122,41 @@ export const resetFilter = ({ pagination }) => dispatch => {
     .then(() => dispatch(loaderActions.stopAIServiceListLoader))
     .catch(() => dispatch(loaderActions.stopAIServiceListLoader));
 };
+
+const fetchFeedbackAPI = (username, orgId, serviceId, token) => {
+  const apiName = APIEndpoints.USER.name;
+  const path = `${APIPaths.FEEDBACK}?username=${username}&org_id=${orgId}&service_id=${serviceId}`;
+  const myInit = {
```
Comment (raamb) [inline src/Redux/actionCreators/ServiceActions.js]: Can we have one method that constructs myInit? The method can take in header values and body value and return this object. I see that its used in other API calls as well

File: src/Redux/actionCreators/ServiceActions.js (hunk 1 above)
Comment (Vivek205) [reply src/Redux/actionCreators/ServiceActions.js]: resolved in [commit](https://github.com/singnet/refactored-dapp/pull/109/commits/b5100b7367d4ad7de42f0c97f844a6ad682f316a)

File: src/Redux/actionCreators/ServiceActions.js (hunk 2 above)
Comment (Vivek205) [reply src/Redux/actionCreators/ServiceActions.js]: resolved in [commit](https://github.com/singnet/refactored-dapp/pull/109/commits/56e18282bd7504a6989b5ea1d22e9c4b581a313f)

**Pull request merged**